docs/DafnyRef/Attributes.md

```diff
@@ -141,32 +95,96 @@ be added.
    if (F != null) { Repr := Repr + {F} + F.Repr; }
 ```
 
-### 22.1.4. axiom
+### 22.1.2. `{:nativeType}` {#sec-nativetype}
+The `{:nativeType}` attribute may only be used on a ``NewtypeDecl``
+where the base type is an integral type. It can take one of the following
+forms:
+
+* `{:nativeType}` - With no parameters it has no effect and the ``NewtypeDecl``
+have its default behavior which is to choose a native type that can hold any
+value satisfying the constraints, if possible, otherwise BigInteger is used.
+* `{:nativeType true}` - Also gives default ``NewtypeDecl`` behavior,
+but gives an error if base type is not integral.
+* `{:nativeType false}` - Inhibits using a native type. BigInteger is used
+for integral types and BitRational for real types.
+* `{:nativeType "typename"}` - This form has an native integral
+type name as a string literal. Acceptable values are: "byte",
+"sbyte", "ushort", "short", "uint", "int", "ulong" and "long".
+An error is reported if the given datatype cannot hold all the
+values that satisfy the constraint.
+
+### 22.1.3. `{:ignore}`
+Ignore the declaration (after checking for duplicate names).
```

Comment:
Do the verifier and compilers respect this attribute? It seems like a really strange attribute to me. I don't know (or remember) where it came from, but I would recommend eliminating support for this attribute (in a different PR would be fine).

Reply:
Boogie does verify this attribute, and it looks like extern trigger some `{:ignore}`. I'll mark it as deprecated